src/content/api/compilation-hooks.md

```diff
@@ -464,6 +465,50 @@ The assets have been optimized.
 - Callback Parameters: `assets`
 
 
+### `processAssets`
+
+`AsyncSeriesHook`
+
+Asset processing.
+
+- Callback Parameters: `assets`
+
+Here's an example:
+
+```js
+compilation.hooks.processAssets.tap(
+  {
+    name: 'MyPlugin',
+    stage: Compilation.PROCESS_ASSETS_STAGE_ADDITIONS,
+  },
+  (assets) => {
+    delete assets['a.js'];
+  }
+);
+```
+
+There're many stages to use:
+
+- `PROCESS_ASSETS_STAGE_ADDITIONAL` - Add additional assets to the compilation.
```

Comment:
Defined in `types.d.ts` https://github.com/webpack/webpack/blob/a4d06805f2f934132ee14229a73aaa5c532a822b/types.d.ts#L1214